### PR TITLE
feat(bundle): Optimizing bundle size

### PR DESCRIPTION
**Pull request by jarjan**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] All tests are passing
- [ ] New/updated tests are included
- [x] Run the precommit hook (it's going to run all required actions such as `Prettier`, `Eslint`, `Jest` and so on...)

**Other information:**
Upgraded `rollup` and added bundle analyzer which is created on each build, can be found in `statistics.html` at root.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
     "moment-timezone": "^0.5.16",
     "normalize.css": "^8.0.0",
     "polished": "^1.9.2",
-    "react": "16.6.0",
-    "react-dom": "16.6.0",
+    "react": "16.6.3",
```
Comment (maroun-baydoun) [inline package.json]: @jarjan Do you think we should declare these `react` packages inside `devDependencies` instead? The only time they will be needed is when doing testing etc. In other scenarios they should be resolved by the `peerDependencies`.

**Comment by vinicius33** [comment]
That's so cool!

File: package.json (hunk 1 above)
Comment (jarjan) [reply package.json]: @maroun-baydoun yes, that makes sense!

**Pull request merged**